shacl/index.html

```diff
@@ -491,14 +491,14 @@
 				</p>
 				<p>
 					Note that <code>rdfs:subClassOf</code> transitivity is only used when SHACL explicitly determines type and subclass relationships in the shapes graph and the data.
-					In other places only triples that are explicitly present in the shapes graph or the data are considered.
+					Elsewhere, only triples that are explicitly present in the shapes graph or the data are considered.
 					For example, SHACL property constraints on <code>rdf:type</code> and <code>rdfs:subClassOf</code> only utilize triples that are explicitly in the data.
 					Similarly, subproperties of <code>rdfs:label</code> and <code>rdfs:commment</code> and subclasses of <code>rdf:List</code> are not recognized in the shapes graph.
 				</p>
 				<p>
 					These design decisions mean that SHACL processors do not have to natively support full RDFS inferencing.
-					However, SHACL validation engines may operate on RDF graphs that include entailments - either pre-computed before being submitted to a SHACL processor or performed on the fly as part of SHACL processing.
-					To support on the fly processing of entailments, SHACL includes the property <a href="#entailment"><code>sh:entailment</code></a> to indicate what kind of inferencing is required by a given shapes graph.
+					However, SHACL validation applications may operate on RDF graphs that include entailments - either pre-computed before being submitted to a SHACL processor or performed on the fly as part of SHACL processing.
```

Comment:
SHACL is not an agent, so SHACL cannot act. Nor can a shape. So we need 
a way to say what it is that is *doing*. There isn't a definition of 
"SHACL validation engine" either, so perhaps we need to do something in 
the introduction to introduce "SHACL validation application". The other 
option is that we try to leave off reference to "SHACL 
processors/validation engines/applications" where we can. I'll read 
through again and see if I can come up with something.

kc

On 4/24/16 6:46 AM, Dimitris Kontokostas wrote:
> In shacl/index.html
> <https://github.com/w3c/data-shapes/pull/18#discussion_r60844837>:
>
>>  					For example, SHACL property constraints on <code>rdf:type</code> and <code>rdfs:subClassOf</code> only utilize triples that are explicitly in the data.
>>  					Similarly, subproperties of <code>rdfs:label</code> and <code>rdfs:commment</code> and subclasses of <code>rdf:List</code> are not recognized in the shapes graph.
>>  				</p>
>>  				<p>
>>  					These design decisions mean that SHACL processors do not have to natively support full RDFS inferencing.
>> -					However, SHACL validation engines may operate on RDF graphs that include entailments - either pre-computed before being submitted to a SHACL processor or performed on the fly as part of SHACL processing.
>> -					To support on the fly processing of entailments, SHACL includes the property <a href="#entailment"><code>sh:entailment</code></a> to indicate what kind of inferencing is required by a given shapes graph.
>> +					However, SHACL validation applications may operate on RDF graphs that include entailments - either pre-computed before being submitted to a SHACL processor or performed on the fly as part of SHACL processing.
>
> we did not define so far what a "SHACL validation application" is. What
> about:
> "However, SHACL may operate on RDF graphs..."?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/w3c/data-shapes/pull/18/files/52fded8308a039314d783a385110ed702c951dcb#r60844837>
>

-- 
Karen Coyle
kcoyle@kcoyle.net http://kcoyle.net
m: 1-510-435-8234
skype: kcoylenet/+1-510-984-3600